server/gateway/src/app.ts

```diff
@@ -192,7 +194,9 @@ export function create(
                         ? process.env.MSA_CONFIGURATION_CLIENT_ID : msaConfiguration.clientId,
                     clientSecret: _.isEmpty(msaConfiguration.secret)
                         ? process.env.MSA_CONFIGURATION_CLIENT_SECRET : msaConfiguration.secret,
-                    issuer: "https://login.microsoftonline.com/9188040d-6c67-4c5b-b112-36a304b66dad/v2.0",
+                    issuer: _.isEmpty(process.env.SP_ISSUER)
+                        ?  "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0"
```

Comment:
Was the guid supposed to change?

Reply:
The MS issuer should stay the same, changed it back. Thanks!